src/test/app/claims/converters/claimantResponseConverter.ts

```diff
@@ -0,0 +1,228 @@
+        {
+          'type': 'acceptation',
+          'amountPaid': 10,
+          'claimantPaymentIntention': {
+            'paymentOption': 'IMMEDIATELY'
+          },
+          'courtDetermination': {
+            ...courtDecisionInstalments,
+            'disposableIncome': 200
+          },
+          'formaliseOption': 'CCJ',
+          'decisionType': 'COURT'
+        })
+    })
+
+    it(' Accept court decision favouring defendant payment intent ', () => {
+      expect(converter.covertToClaimantResponse(createDCRWithCourtDecisionType(intentionOfPaymentInFullBySetDate,DecisionType.DEFENDANT,FormaliseRepaymentPlanOption.REQUEST_COUNTY_COURT_JUDGEMENT))).to.deep.eq(
+        {
+          'type': 'acceptation',
+          'amountPaid': 10,
+          'claimantPaymentIntention': {
+            'paymentOption': 'BY_SPECIFIED_DATE',
+            'paymentDate': new LocalDate(2018,12,31).toMoment()
+          },
+          'courtDetermination': {
+            ...courtDecisionInstalments,
+            'disposableIncome': 200
+          },
+          'formaliseOption': 'CCJ',
+          'decisionType': 'DEFENDANT'
+        })
+    })
+
+    it(' Accept court decision favouring claimant payment intent ', () => {
+      expect(converter.covertToClaimantResponse(createDCRWithCourtDecisionType(intentionOfPaymentByInstallments,DecisionType.CLAIMANT,FormaliseRepaymentPlanOption.REQUEST_COUNTY_COURT_JUDGEMENT))).to.deep.eq(
+        {
+          'type': 'acceptation',
+          'amountPaid': 10,
+          'claimantPaymentIntention': {
+            'paymentOption': 'INSTALMENTS',
+            'repaymentPlan': {
+              'firstPaymentDate': new LocalDate(2018,12,31).toMoment(),
+              'instalmentAmount': 100,
+              'paymentSchedule': 'EVERY_MONTH'
+            }
+          },
+          'courtDetermination': {
+            ...courtDecisionInstalments,
+            'disposableIncome': 200
+          },
+          'formaliseOption': 'CCJ',
+          'decisionType': 'CLAIMANT'
+        })
+    })
+
+    it(' Reject court decision and refer to judge ', () => {
+      const dcr = createDCRWithCourtDecisionType(intentionOfImmediatePayment,DecisionType.COURT,FormaliseRepaymentPlanOption.REFER_TO_JUDGE)
+      dcr.rejectionReason = new RejectionReason('rejected reason')
+      expect(converter.covertToClaimantResponse(dcr)).to.deep.eq(
+        {
+          'type': 'acceptation',
+          'amountPaid': 10,
+          'claimantPaymentIntention': {
+            'paymentOption': 'IMMEDIATELY'
+          },
+          'courtDetermination': {
+            ...courtDecisionInstalments,
+            'disposableIncome': 200,
+            'rejectionReason': 'rejected reason'
+          },
+          'formaliseOption': 'REFER_TO_JUDGE',
+          'decisionType': 'COURT'
+        })
+    })
+
```

Comment:
Would be nice to have two more negative tests here:

1. When 'decisionType': 'COURT' and `courtDetermination` is not in draft
2. When 'decisionType': 'CLAIMANT' and `claimantPaymentIntention ` is not in draft

Reply:
added two more tests